Header header
BoundingBox[] bounding_boxes
sensor_msgs/Image frame